tools/check-api.js

```diff
@@ -53,8 +53,7 @@ class API {
         console.log(error)
       })
     }
-
-    fs.writeFile(file, outLines.join('## '), () => {})
+    fs.writeFileSync(file, outLines.join('## '), () => {})
```

Comment:
We can remove the callback: `fs.writeFileSync(file, outLines.join('## '))`

Reply:
done